felix/rules/static.go

```diff
@@ -1144,40 +1144,53 @@ func (r *DefaultRuleRenderer) StaticBPFModeRawChains(ipVersion uint8,
 		Rules: rawRules,
 	}
 
-	bpfUntrackedFlowChain := &Chain{
-		Name:  ChainRawUntrackedFlows,
-		Rules: []Rule{},
-	}
+	// BPF Untracked Flow Rules:
+	var bpfUntrackedFlowRules []Rule
+	var bpfUntrackedFlowChain *Chain
```

Comment:
Minor: as a general rule, it's best to minimise scope of variables and I don't think bpfUntrackedFlowChain is used until line 1191 so may as well declare it there.

Reply:
Updated as recommended